openhtf/io/output/print_console.py

```diff
@@ -0,0 +1,91 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+
+
+class PrintConsole():
+  """Print test results with failur info on console. """
```

Comment:
s/failur/failure

Maybe something like:
`Print test outcome (including any failed measurements) to the console.`